README.md

```diff
@@ -4,7 +4,7 @@
 
 The front end application for Red Hat iPaaS - a flexible, customizable, cloud-hosted platform that provides core integration capabilities as a service. It leverages Red Hat's existing product architecture using OpenShift Online/Dedicated and Fuse Integration Services.
 
-For the middle tier API that this client communicates with, please see [this](https://github.com/redhat-ipaas/ipaas-api) repo. 
+For the middle tier API that this client communicates with, please see [this](https://github.com/redhat-ipaas/ipaas-api) repo.
```

Comment:
I thought I had updated this to reflect the actual path `https://github.com/redhat-ipaas/ipaas-api-java`. @KurtStam said he was fine with keeping the name as is, so I didn't delete the original API repo, but still can if either of you change your mind. Until then, this should be updated.